WORKSPACE

```diff
@@ -14,170 +14,30 @@
 
 workspace(name = "io_opencensus_cpp")
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load(":deps.bzl", "opencensus_cpp_deps")
 
-# Build rules for C++ projects.
-http_archive(
-    name = "rules_cc",
-    strip_prefix = "rules_cc-master",
-    urls = ["https://github.com/bazelbuild/rules_cc/archive/master.zip"],
-)
-
-# We depend on Abseil.
-http_archive(
-    name = "com_google_absl",
-    strip_prefix = "abseil-cpp-master",
-    urls = ["https://github.com/abseil/abseil-cpp/archive/master.zip"],
-)
-
-# GoogleTest framework.
-# Only needed for tests, not to build the OpenCensus library.
-http_archive(
-    name = "com_google_googletest",
```

Comment:
googletest is not used for building, only for testing, so IMO it should be in WORKSPACE directly, and not part of `opencensus_cpp_deps()`

Ditto benchmark.